L/libadwaita/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "libadwaita"
+version = v"1.2.0"
+
+sources = [
+    ArchiveSource("https://download.gnome.org/sources/libadwaita/$(version.major).$(version.minor)/libadwaita-$(version).tar.xz",
+                  "322f3e1be39ba67981d9fe7228a85818eccaa2ed0aa42bcafe263af881c6460c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+# We need to run some commands with a native Glib
+apk add glib-dev
+
+# Copied from GTK4 recipe
+ln -sf /usr/bin/glib-compile-resources ${bindir}/glib-compile-resources
+ln -sf /usr/bin/glib-compile-schemas ${bindir}/glib-compile-schemas
+ln -sf /usr/bin/gdk-pixbuf-pixdata ${bindir}/gdk-pixbuf-pixdata
+# Remove gio-2.0 pkgconfig file so that it isn't picked up by post-install script.
+rm ${prefix}/lib/pkgconfig/gio-2.0.pc
+
+# oldest version that works, due to use of "@available" macro
+export MACOSX_DEPLOYMENT_TARGET=10.14
+
+cd libadwaita*
+
+mkdir build-libadwaita && cd build-libadwaita
+meson .. \
+    --buildtype=plain \
```

Comment:
Why plain and not release?

Reply:
I can't remember how I decided to try `buildtype=plain`, since I started this months ago and picked it up again today. But on i686-w64-mingw32, when I use `buildtype=release`, I'm getting:
```
[123/123] Linking target src/libadwaita-1-0.dll
[19:44:21] ninja: job failed: ...
[19:44:21] ninja: subcommand failed
[19:44:21] src/libadwaita-1-0.dll.p/adw-leaflet.c.obj:adw-leaflet.c:(.text+0xca5): undefined reference to `__stack_chk_guard'
```

Reply:
Sigh.  Ok, I don't know what to do with that (I presume we need to link some mingw libraries, but don't know which), so let's stick to plain for the time being.

Reply:
Thanks